grails-app/controllers/org/pih/warehouse/RoleInterceptor.groovy

```diff
@@ -140,9 +141,12 @@ class RoleInterceptor {
     }
 
     static Boolean needSuperuser(controllerName, actionName) {
-        superuserControllers?.contains(controllerName) || superuserActions[controllerName]?.contains(actionName) || superuserActions['*'].any {
-            actionName?.startsWith(it)
-        }
+        (superuserActions[controllerName]?.contains("*")
+            || superuserControllers?.contains(controllerName)
+            || superuserActions[controllerName]?.contains(actionName)
+            || superuserActions['*'].any {
+                actionName?.startsWith(it)
+            })
```

Comment:
```suggestion
            || superuserActions['*'].any(actionName?.startsWith(it))
            )
```
I think we can make it a little more compact without having to use `{ ... }` but it doesn't really matter I am fine with the way it looks.

Reply:
I agree, I have not realized that, because this code has been already there, the only thing I've introduced here is:
```groovy
superuserActions[controllerName]?.contains("*")
```
I could change it

Reply:
@drodzewicz I've applied your suggestion, but in fact, it should not be `any(...)`, but `any{...}`.

Reply:
by saying so I'm starting to to strongly agree with @awalkowiak that applying suggestions blindly via github sucks.